packages/plugin-ext/src/api/plugin-api.ts

```diff
@@ -39,6 +39,7 @@ import {
     DefinitionLink,
     DocumentLink
 } from './model';
+import {CancellationToken} from '@theia/plugin';
```

Comment:
there is an import
```
 import * as theia from '@theia/plugin';
```

so you should use theia.CancellationToken (without adding this import)